src/table/boot.rs

```diff
@@ -236,6 +236,11 @@ impl BootServices {
         assert_eq!((self.stall)(time), Status::Success);
     }
 
+    /// Set the watchdog timer.
+    pub fn set_watchdog_timer(&self, timeout: usize, watchdog_code: u64, data_size: usize, watchdog_data: *mut u16) {
+        assert_eq!((self.set_watchdog_timer)(timeout, watchdog_code, data_size, watchdog_data), Status::Success);
```

Comment:
This method can return status codes other than Success if the wrong parameters are passed in, therefore a Result should be propagated.

Reply:
@GabrielMajeri Already applied some changes on `master` :)

Reply:
Gotta go fast! :smile: